src/lib/server/database/index.test.ts

```diff
@@ -125,6 +125,8 @@ it('should complete a full user journey', async () => {
         expect(await db.canEditTicketTitle(tid, uid)).toStrictEqual(true);
         // TODO: add test case when the agent actually does have permission
         expect(await db.editTicketTitle(tid, 'New Title')).toStrictEqual(true);
+
+        expect(await db.editTicketDueDate(tid, new Date(Date.now() + 1))).toStrictEqual(true);
```

Comment:
Provide test cases for:

* Non-existent Ticket ID
* Date is behind `Date.now()`